kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/data/RecordConverter.java

```diff
@@ -126,10 +127,14 @@ private Object convertValue(
       case STRING:
         return convertString(value);
       case UUID:
+        if ("parquet".equals(config.writeProps().get("write.format.default"))) {
+          return UUIDUtil.convert(convertUUID(value));
```

Comment:
We could move the call to `UUIDUtil.convert()` into the `convertUUID()` method I think.